zlib.h

```diff
@@ -1782,7 +1782,7 @@ ZEXTERN int ZEXPORT gzgetc_(gzFile file);  /* backward compatibility */
    ZEXTERN void ZEXPORT crc32_combine_gen64(uint32_t *op, z_off64_t);
 #endif
 
-#if !defined(ZLIB_INTERNAL) && defined(Z_WANT64)
+#if !defined(ZINTERNAL) && defined(Z_WANT64)
```

Comment:
I am unsure about this change here. It is in zlib.h. Perhaps we should leave this one to `ZLIB_INTERNAL`.

Reply:
This basically checks whether we are currently compiling zlib(-ng) or compiling an external application. So the define itself should never come from an external source. Therefore I think this is fine.